server/src/main/java/org/opensearch/search/SearchHit.java

```diff
@@ -164,6 +169,9 @@ public SearchHit(
         this.nestedIdentity = nestedIdentity;
         this.documentFields = documentFields == null ? emptyMap() : documentFields;
         this.metaFields = metaFields == null ? emptyMap() : metaFields;
+        if (FeatureFlags.isEnabled(FeatureFlags.PROTOBUF)) {
```

Comment:
The model classes should not use `FeatureFlags`

Reply:
Right, I missed moving this.